work-scheduler-service/src/main/java/com/github/gaalgergo/workscheduler/service/employees/EmployeeServiceImpl.java

```diff
@@ -28,4 +29,10 @@
             .map(entity -> conversionService.convert(entity, Employee.class))
             .collect(Collectors.toList());
     }
+
+    @Override
+    public void save(Employee employee) {
+        employeeRepository.save(conversionService.convert(employee, EmployeeEntity.class));
```

Comment:
You don't have a `Converter` implementation for converting `Employee` instances to `EmployeeEntity` instances, only the other way around.
Create a new `Converter` for this.